parlai/mturk/core/agents.py

```diff
@@ -58,7 +60,14 @@ def init_aws(self, opt):
             return
 
         print('Setting up MTurk backend...')
-        html_api_endpoint_url, json_api_endpoint_url, requester_key_gt = setup_aws(task_description=opt['task_description'], num_hits=opt['num_hits'], num_assignments=opt['num_assignments'], is_sandbox=opt['is_sandbox'])
+        create_hit_config(task_description=opt['task_description'], num_hits=opt['num_hits'], num_assignments=opt['num_assignments'], is_sandbox=opt['is_sandbox'])
+        if not self.task_files_to_copy:
```

Comment:
@yf225 shouldn't you check here if each mturk_agent_id has its templates in task_files_to_copy?